tms-grpc-service/src/controllers/healthcheck.ts

```diff
@@ -0,0 +1,48 @@
+import { Context, Next } from 'koa';
+import { HealthCheckRequest } from '../../models/health_pb';
+import { healthService } from '../clients/health';
+import { config } from '../config';
+import { logger } from '../utils';
+
+export const healthCheck = async (ctx: Context, next: Next): Promise<void | Context> => {
+  try {
+    const param: HealthCheckRequest = new HealthCheckRequest();
+    param.setService(config.serviceName);
+
+    const res = await healthService.check(param);
```

Comment:
Are we spinning up a gRPC Health Server, just so we can call it from inside the same NodeJS application? What's the purpose of this?